docker-compose.yml

```diff
@@ -17,7 +17,9 @@ services:
       - mysql
     volumes:
       - ${PWD}:/opt/coding-challenge-api
-    command: /bin/bash -c "sleep 25; rake db:drop db:create db:migrate && rails server -b 0.0.0.0"
+    command: /bin/bash -c "sleep 25; rake db:drop db:create db:migrate && rm tmp/pids/server.pid && rails server -b 0.0.0.0"
```

Comment:
sometimes we stop the execution of `docker-compose up` with `Ctrl+C` and the file `tmp/pids/server.pid` is not removed from the container, so it is not possible to start again `docker-compose up` if we don't get into the container to remove this file. This new `rm` command will remove this file whether it exists.